zigbeetradfri/README.md

```diff
@@ -8,12 +8,12 @@ This plugin allows to interact with IKEA TRÅDFRI ZigBee devices using a native
 * TRADFRI bulb E27 WS clear 806lm (color temperature light)
 * [Wireless dimmer](https://www.ikea.com/us/en/p/tradfri-wireless-dimmer-white-10408598/)*
 * [Shortcut Button](https://www.ikea.com/us/en/p/tradfri-shortcut-button-white-20356382/)*
-* [Remote control](https://www.ikea.com/us/en/p/tradfri-remote-control-00443130/)**
+* [Remote control](https://www.ikea.com/us/en/p/tradfri-control-outlet-kit-smart-70364803/)**
```

Comment:
This seems wrong. The remote control is the one shown on this picture: https://www.ikea.com/us/en/customer-service/product-support/smart-lighting/

Looks like IKEA stopped producing this one though, so there isn't a proper link for this one any more on the IKEA website. Perhaps we could use this one instead: https://zigbee.blakadder.com/Ikea_E1810.html